development.cfg

```diff
@@ -1,4 +1,4 @@
 [buildout]
 extends =
-    test-plone-5.1.x.cfg
+    test-plone-4.3.x.cfg
```

Comment:
Please don't change back to 4.3 :-)

Reply:
Sorry, I changed that for local plone 5 tests and sneaked it in.